docs/reference/release-notes/7.0.0.asciidoc

```diff
@@ -1,8 +1,1648 @@
 [[release-notes-7.0.0]]
 == {es} version 7.0.0
 
-//These release notes include all changes made in the alpha, beta, and RC
-//releases of 7.0.0. 
+These release notes include all changes made in the alpha, beta, and RC
+releases of 7.0.0.
 
-//Also see <<breaking-changes-7.0,Breaking changes in 7.0.0>>.
+Also see <<breaking-changes-7.0,Breaking changes in 7.0>>.
 
+[[breaking-7.0.0]]
+[float]
+=== Breaking changes
+
+Aggregations::
+* Remove support for deprecated params._agg/_aggs for scripted metric aggregations {pull}32979[#32979] (issues: {issue}29328[#29328], {issue}31597[#31597])
+* Percentile/Ranks should return null instead of NaN when empty {pull}30460[#30460] (issue: {issue}29066[#29066])
+* Render sum as zero if count is zero for stats aggregation {pull}27193[#27193] (issue: {issue}26893[#26893])
```

Comment:
I don't see documentation for https://github.com/elastic/elasticsearch/pull/27193 in the Breaking Changes docs